proposal-template.md

```diff
@@ -43,26 +43,61 @@ Three separate groups of users will be involved in this project. The first group
 3 | Provide technicians with route information for their currently planned routes | A mobile interface with routing and itinerary information (which robot to fix next) | Convenience for technicians who need to know their next task | should be able to see locations of robots needing maintenance on a map. should be able to see route information for current user. should be able to check-off schedule items as repaired, or needs parts, etc. (add information to file).
-4 | Consolidate failure & usage data to help develop understanding of assets | A data visualization component of the web-based IOC containing live-updated charts and figures | Improved business knowledge regarding product failure | Bar chart plotting replacement parts used in repairs. Line chart plotting number of robots requiring repairs. At least 4 other meaningful visualizations must be available.
+4 | Consolidate failure & usage data to help develop understanding of assets | A data visualization component of the web-based IOC containing live-updated charts and figures | Improved business knowledge regarding product failure | Bar chart plotting replacement parts used in repairs. Line chart plotting number of robots requiring repairs. At least 4 other meaningful visualizations must be provided.
 5 | Show current and historical spatial state of assets and crew | A map component of the web-based IOC capable of displaying historic and current locations of connected robots, crew, and routes | Improved monitoring and spatial knowledge | filters should be available based on status of robots. should be able to show the current location of maintenance crews. should be able to show current routes planned for crews. should be able to create/ edit routes for crews.
 
-
 > For the measurement criteria provide a statement that describes what the project will accomplish and the performance measure. The project objectives whenever possible should be written to be specific, measurable, achievable, realistic and timely (SMART). The measurement criteria specifies a metric(s) that will measure success.  
 
 ## Project Performance Measurement Plan
 > For each project objective and measurement criteria identified in the above table, indicate the methods used to measure, who is responsible, frequency of measurement, key targets and reporting on results. For example, if there are three(3) project objectives, fill out a performance measurement plan for each objective.
 
 Project Objective | Measurement Criteria
 ------------ | -------------
-**Description of Measure** | **Method**
-Method used to measure |
-Responsibility: who measures/analyses/reports|
-Frequency of measure |
-Key targets |
-Reporting on results |
+**Suggest routes for crews** | **Method**
+Method used to measure | Comparison of alogirithm on random data sets 
+Responsibility: who measures/analyses/reports| Lucas Hossack
```

Comment:
Should this be analyzes? Instead of analyses? Same with all the other ones

Reply:
They should all be analyzes

Reply:
This was from the template doc..